### PR TITLE
UBERF-6677: add user online/offline status

### PR DESCRIPTION
**Pull request by kristina-fefelova**
Places with user status

Direct:
Screenshot from 2024-04-23 17-22-09
Screenshot from 2024-04-23 17-56-42

Direct header:
Screenshot from 2024-04-23 17-53-45

Search users modal when creating direct or add users to channels:
Screenshot from 2024-04-23 18-13-48

Channel members:
Screenshot from 2024-04-23 17-54-57

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjI3ZDI3MDVmYTEwNWM0NjI5YmU2MDQiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.PPvmVSEQ0RoVn7K-u_VIkD3ZtzTD5d3qiUIAaz2jOhs">Huly&reg;: <b>UBERF-6681</b></a></sub>

### CODE REVIEW

File: plugins/view-resources/src/components/UserStatus.svelte [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+  export let user: Ref<Account>
+  export let size: 'small' | 'medium' = 'small'
+  export let background: string = 'red'
+
+  const query = createQuery()
+
+  let status: UserStatus | undefined = undefined
+
+  $: query.query(
```
Comment (aonnikov) [inline plugins/view-resources/src/components/UserStatus.svelte]: Just an idea, maybe put statuses into svelte store so we do not query them?

File: plugins/contact-resources/src/components/Avatar.svelte [hunk 2]
```diff
@@ -364,4 +392,10 @@
       font-size: inherit;
     }
   }
+
+  .status {
+    position: absolute;
+    bottom: -0.125ren;
```
Comment (aonnikov) [inline plugins/contact-resources/src/components/Avatar.svelte]: typo: `ren` -> `rem`

File: plugins/view-resources/src/components/UserStatus.svelte (hunk 1 above)
Comment (kristina-fefelova) [reply plugins/view-resources/src/components/UserStatus.svelte]: Done.

File: plugins/contact-resources/src/components/Avatar.svelte (hunk 2 above)
Comment (kristina-fefelova) [reply plugins/contact-resources/src/components/Avatar.svelte]: Done.

**Review by haiodo — APPROVED** [review]

**Pull request merged**